lib/ui/painting/image_decoder_impeller.cc

```diff
@@ -108,13 +120,13 @@ static sk_sp<DlImage> DecompressAndUploadTexture(
   }
 
   if (!texture->SetContents(
-          reinterpret_cast<const uint8_t*>(bitmap.getAddr(0, 0)),
+          reinterpret_cast<const uint8_t*>(bitmap->getAddr(0, 0)),
           texture_descriptor.GetByteSizeOfBaseMipLevel())) {
     FML_DLOG(ERROR) << "Could not copy contents into Impeller texture.";
     return nullptr;
   }
 
-  texture->SetLabel(label.c_str());
+  texture->SetLabel(impeller::SPrintF("ui.Image(%p)", texture.get()).c_str());
```

Comment:
This will show up in tracing/profiling on low end devices. Can we make it debug only?

Reply:
Sure, the right place to disable these would be to [set this conditional to `false` in the reactor](https://github.com/flutter/engine/blob/50b3a6e7a3f9161058c56e85fc1103f2e2953702/impeller/renderer/backend/gles/reactor_gles.cc#L20). This should disable all label collection and reporting. I haven't measured the performance differences when this is always enabled though.